src/GovUk.Education.ExploreEducationStatistics.Data.Processor/ImportRecoveryHandler.cs

```diff
@@ -53,44 +36,8 @@ public static void CheckIncompleteImports(string connectionString)
 
                 foreach (var entity in resultSegment.Results)
                 {
-                    Console.Out.WriteLine($"Recovering {entity.PartitionKey} : {entity.RowKey}");
-
-                    // If batch was not split then just processing again by adding to pending queue
-                    if (entity.Status.Equals(IStatus.QUEUED)
-                        || entity.Status.Equals(IStatus.PROCESSING_ARCHIVE_FILE)
-                        || entity.Status.Equals(IStatus.STAGE_1))
-                    {
-                        Console.WriteLine($"No data imported for {entity.PartitionKey} : {entity.RowKey} - Import " +
-                                          $"will be re-run");
-                        pendingQueue.AddMessage(new CloudQueueMessage(entity.Message));
-                    }
-                    // Else create a message for all remaining batches
-                    else
-                    {
-                        Console.WriteLine($"Creating messages for remaining batches for {entity.PartitionKey} : {entity.RowKey}");
-                        var m = JsonConvert.DeserializeObject<ImportMessage>(entity.Message);
-                        var batches = GetBatchesRemaining(blobContainer, entity.PartitionKey, m.OrigDataFileName)
-                            .Result;
-
-                        // If no batches then assume it didn't get passed initial validation stage
-                        if (!batches.Any())
-                        {
-                            Console.WriteLine($"No batches created for {entity.PartitionKey} : {entity.RowKey} " +
-                                              $"although batches are necessary to create.  Adding message to the " +
-                                              $"queue to try re-import.");
-                            pendingQueue.AddMessage(new CloudQueueMessage(entity.Message));
-                            continue;
-                        }
-
-                        Console.WriteLine($"Creating {batches.Count} messages for remaining batches for {entity.PartitionKey} : {entity.RowKey}");
-                        
-                        foreach (var folderAndFilename in batches)
-                        {
-                            Console.WriteLine($"Creating message for importing batch file {folderAndFilename.Name} " +
-                                              $"for {entity.PartitionKey} : {entity.RowKey}");
-                            availableQueue.AddMessage(new CloudQueueMessage(BuildMessage(m, folderAndFilename.Name)));
-                        }
-                    }
+                    Console.Out.WriteLine($"Recovering interrupted import for {entity.PartitionKey} : {entity.RowKey}");
```

Comment:
Looks nice to be able to remove so much code :smile: wonder why the original implementation was so complicated in comparison?